### ISSUE TITLE
How to change the NormalizedStartTime after the state for the transition is being created

### ISSUE THREAD

**Issue by ericwang0701** [KybernetikGames/animancer#106]
Hi,

I'd like to know if there is any way that I can control the start time of animation in run time after we create an `AnimancerState`. I know we can do it in the editor through `ClipTransition` but if I create a AnimancerState like this:

```
        clipStateTransition.NormalizedStartTime = 0.25f;
        var state =  animancerComponenet.Play(clipStateTransition);
```

It seems we can't change the start time of the animation in the state afterward.

**Comment by KybernetikGames** [comment]
`clipStateTransition.Transition.NormalizedStartTime = ...` to set it on the transition.

Or `state.NormalizedTime = ...` if you just want to set the state's time without modifying the transition.

**Comment by ericwang0701** [comment]
Thank you for your quick reply. I did try to change the start time of the animation by `clipStateTransition.Transition.NormalizedStartTime = ...` but it didn't work

After we create AnimancerState from a clipStateTransition by `var state =  animancerComponenet.Play(clipStateTransition);`, if we modify the `clipStateTransition` (e.g. change speed, start time), will it apply directly to the animation played by the `state `?

Basically, I want to let users change the start and end time of animation in runtime. Any suggestions or sample code on how to achieve it with ClipStateTransition would be really appreciated. Thanks

**Comment by KybernetikGames** [comment]
A transition is just a collection of details that can be applied to a state when it's played. After that, modifying the transition won't affect the state. So you can either set the `clipStateTransition.Transition.NormalizedStartTime` before calling `Play` or set the `state.NormalizedTime` afterwards.

Events are an exception to that rule though. Modifying the transition's events (including the end time) after playing it will still affect the state.
